
public class ExceptionCentral {
	
	public static void main(String[] args) {
		
		secondMethod();
	}

	static int firstMethod() throws ArithmeticException {
		int numberThatExists = 4 / 0;
		
		return numberThatExists;
	}
	
	@SuppressWarnings("null")
	static char secondMethod() throws NullPointerException {
		String thisShouldWork = null;
		
		return thisShouldWork.charAt(0);
	}
	
}

